docs/faq/differences-with.../fivetran-vs-airbyte.md

```diff
@@ -0,0 +1,27 @@
+# Fivetran vs Airbyte
+
+We wrote an article, “[Open-source vs. Commercial Software: How to Solve the Data Integration Problem](https://airbyte.io/articles/data-engineering-thoughts/open-source-vs-commercial-software-how-to-better-solve-data-integration/),” in which we describe the pros and cons of Fivetran’s commercial approach and Airbyte’s open-source approach. Don’t hesitate to check it out for more detailed arguments. As a summary, here are the differences: 
+
+![](https://lh4.googleusercontent.com/bcNM7lS3u9ijFkQzJJoIIUUBzqYSnlpmBQWArPaBQOc2_1ztmDVeH-51wcK43zNWWOmLW-_SjIajenHPbejjorNrTmI31-v8r9h0gOLiqDXwVmmKKASea4ks2l-Y5BoxuFpWT7cz)
+
+### **Fivetran:**
+
+* **Limited high-quality connectors:** after 8 years in business, Fivetran supports 150 connectors. The more connectors, the more difficult it is for Fivetran to keep the same level of maintenance across all connectors. They will always have a ROI consideration to maintaining long-tailed connectors. 
+* **Pricing indexed on usage:** Fivetran’s pricing is indexed on the number of active rows \(rows added or edited\) per month. Teams always need to keep that in mind and are not free to move data without thinking about cost, as the costs can grow fast. 
+* **Security and privacy compliance:** all companies are subject to privacy compliance laws, such as GDPR, CCPA, HIPAA, etc. As a matter of fact, above a certain stage \(about 100 employees\) in a company, all external products need to go through a security compliance process that can take several months. 
+* **No moving data between internal databases:** Fivetran sits in the cloud, so if you have to replicate data from an internal database to another, it makes no sense to have the data move through them \(Fivetran\) for privacy and cost reasons. 
+
+### **Airbyte:**
+
+* **Free, as open source, so no more pricing based on usage**: learn more about our [future business model](https://docs.airbyte.io/company-handbook/company-handbook/business-model) \(connectors will always remain open source\). 
+* **Supporting 50+ connectors by the end of 2020** \(so in only 5 months of existence\). Current goal is to reach 400+ connectors by the end of 2021. 
```

Comment:
why is this one different from the same bullet point in the meltano one?

Reply:
Will update it, I need to update the links, it seems. Thanks!